config/metadata/configMeta.yaml

```diff
@@ -1043,15 +1043,13 @@ groups:
         firstversion: v2.6
         type: int
         valuetype: nondefault
-        default: 5
+        default: 30
```

Comment:
When I ran with 30 connections, the `redisstore_conn_wait` is still pretty high. Maybe we need even more connections available https://ui-dogfood.honeycomb.io/kent_test/environments/thenewworld/datasets/refinery_metrics/result/zGi8XYhTKpE